subsys/bluetooth/host/gatt.c

```diff
@@ -3919,6 +3919,13 @@ static uint16_t parse_include(struct bt_conn *conn, const void *pdu,
 		struct bt_uuid_128 u128;
 	} u;
 
+	if (length < sizeof(*rsp)) {
+		LOG_WRN("Parse err %s", __func__);
```

Comment:
> I don't find the `__func__` necessary. 
> 
> 
> 
> If a user wants that information, it would be a simple matter of enabling `CONFIG_LOG_FUNC_NAME_PREFIX_WRN=y`

Agree. We should avoid such redundant info in the log string, which the log subsystem itself is able to provide when enabled.

Reply:
:+1: Removed use of `__func__`.